kong/conf_loader/init.lua

```diff
@@ -986,6 +1002,9 @@ local function overrides(k, default_v, opts, file_conf, arg_conf)
     value = default_v == "NONE" and "" or default_v
 
   else
+    if not file_conf then
```

Comment:
Hmm, I might be missing something, but what is this addition for?

Reply:
Leftover from debugging. Thanks!